### PR TITLE
US125733/add download action for attachments.

### PR DESCRIPTION
**Pull request by mbosiljevac**

### CODE REVIEW

File: components/left-panel/discussions/consistent-evaluation-discussion-evidence-body.js [hunk 1]
```diff
@@ -145,10 +150,18 @@ export class ConsistentEvaluationDiscussionEvidenceBody extends RtlMixin(Localiz
...
-				const { name, size } = attachment.properties;
+				const { name, size, href } = attachment.properties;
+				const onClickHandler = () => this._downloadAttachment(href);
+				const onKeydownHandler = (e) => {
+					if (e.key === 'Enter' || e.key === ' ') {
+						this._downloadAttachment(href);
+					}
+				};
 
 				return html`<d2l-list-item
```
Comment (amauryperez) [inline components/left-panel/discussions/consistent-evaluation-discussion-evidence-body.js:161]: should we ask Fiona if it makes sense to add some alt text to the link to clarify it performs a download? and also for accessibility?

**Review by amauryperez — APPROVED** [review]

File: components/left-panel/discussions/consistent-evaluation-discussion-evidence-body.js (hunk 1 above)
Comment (mbosiljevac) [reply components/left-panel/discussions/consistent-evaluation-discussion-evidence-body.js:161]: Good idea! I'll send her a message!

**Review by JamieTran — APPROVED** [review]
lgtm!

File: components/left-panel/discussions/consistent-evaluation-discussion-evidence-body.js (hunk 1 above)
Comment (mbosiljevac) [reply components/left-panel/discussions/consistent-evaluation-discussion-evidence-body.js]: chatted with Fiona and added an aria-label that specifies that clicking the link downloads the file

File: lang/en.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ export default {
 	"averageMessageScore": "Average post score",
 	"backToUserSubmissions": "Back to User Submissions",
 	"cancelBtn": "Cancel",
+	"clickToDownloadFile": "click to download {fileName}",
```
Comment (amauryperez) [inline lang/en.js:18]: uppercase? `Click`

**Review by amauryperez — APPROVED** [review]

**Pull request merged**